### PR TITLE
Facade draft

### PR DESCRIPTION
**Pull request by ParkerLockhart**
Done on this branch: 
- [x] Wrote Tests 
- [x] Implemented 
- [x] Reviewed

Requirements for merge:
- [x] All Tests are Passing (If not, please explain below)
- [x] The code will run locally (If not, please explain below)

Type of change:
- [x] New feature
- [ ] Bug Fix

Check the applicable box(es):
- [x] This broke nothing
- [ ] This broke some stuff (Please explain what is broken, include error messages/important info)
- [ ] This broke everything (Please describe what in the world happened)

Testing changes:
- [x] No tests have been changed
- [ ] Some tests have been changed (Please explain what needed to be changed and why)
- [ ] All of the tests have been changed (Please describe what in the world happened)

Cleanup & dev empathy checklist:
- [x] My code has no unused/commented out code
- [x] I double-checked for stray binding.pry and save_and_open_page
- [ ] I commented my code in hard-to-understand areas

### CODE REVIEW

File: app/services/movie_service.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  def movie_info(movie_id)
+    get_url("/movie/#{movie_id}?api_key=#{ENV['movie_api_key']}&language=en-US")
+  end
+
+  def top_rated
+    get_url("/movie/top_rated?api_key=#{ENV['movie_api_key']}&language=en-US")[:results]
+  end
+
+  def get_url(url)
+    response = Faraday.get("https://api.themoviedb.org/3#{url}")
```
Comment (LelandCurtis) [inline app/services/movie_service.rb:26]: we coudl refactor to add the api key here. I don't think it matters what order the query parameters are in , so long as it's after the. '?'. We could add the api= param to the end of every url and remove that section from the other definitions.

Comment (ParkerLockhart) [reply app/services/movie_service.rb:26]: That's a good point! I hadn't thought of if the order matters or not.

File: spec/features/users/movies_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'user movie results page' do
+  let!(:user) {User.create(name: "Jeff", email: "jeff@email.com")}
+  let!(:top_rated_url) {"/users/#{user.id}/movies?q=top%20rated"}
+
+  scenario 'user top rated movies', :vcr do
```
Comment (LelandCurtis) [inline spec/features/users/movies_spec.rb:7]: does this `:vcr` eliminate the need to wrap things in a VCR.use_cassette block?

**Review by LelandCurtis — APPROVED** [review]
Looks amazing.

**Pull request merged**